.babelrc

```diff
@@ -17,13 +20,36 @@
           "public": "./public"
         }
       }
-    ],
+    ]
   ],
   "env": {
     "development": {
       "plugins": [
         "react-hot-loader/babel"
       ]
+    },
+    "test": {
+      "presets": [
+        ["es2015"],
+        "react",
+        "stage-0"
+      ],
+      "plugins": [
+        "transform-runtime",
+        "transform-decorators-legacy",
```

Comment:
Yeap, just because of:

<img width="1072" alt="screen shot 2017-11-23 at 10 38 46" src="https://user-images.githubusercontent.com/6247404/33164228-8f53b5aa-d03a-11e7-8f55-4cfb54a20475.png">